databricks/koalas/tests/test_series_datetime.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2019 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+
+import numpy as np
+import pandas as pd
+
+from databricks import koalas
+from databricks.koalas.testing.utils import ReusedSQLTestCase, TestUtils
+
+
+class SeriesDatetimeTest(ReusedSQLTestCase, TestUtils):
+
+    @property
+    def pdf1(self):
```

Comment:
are these tests coming from the pandas test suites?

Reply:
These tests were actually from the blog that will be published soon.